frames/src/main/java/com/checkout/frames/utils/extensions/BillingAddressDetailsExtensions.kt

```diff
@@ -0,0 +1,107 @@
+package com.checkout.frames.utils.extensions
+
+import androidx.annotation.StringRes
+import com.checkout.base.model.Country
+import com.checkout.frames.R
+import com.checkout.frames.component.billingaddressfields.BillingAddressInputComponentState
+import com.checkout.frames.screen.billingaddress.billingaddressdetails.models.BillingAddress
+import com.checkout.frames.screen.billingaddress.billingaddressdetails.models.BillingFormFields
+import com.checkout.tokenization.model.Phone
+
+internal fun List<BillingAddressInputComponentState>.saveBillingAddressDetails(country: Country): BillingAddress {
```

Comment:
Looks like the function provides `BillingAddress` but not saves it.

Reply:
Updated it to provideBillingAddressDetails.